src/dao.ts

```diff
@@ -11,6 +11,7 @@ import {
 import { VOTES_MANAGER_ENTITY_ID } from "./CONSTANTS";
```

Comment:
Commented out all functionality in this file. Not used in production. Was a bad merge request.